diagnostics/model_accuracy/model-run.json

```diff
@@ -0,0 +1,16 @@
+[
```

Comment:
What happened to entries for models "MobileNet_v1" and "SSD-MobileNet_v1"?  They were present in the original JSON list (in the python script), but are missing in this external JSON file.

Reply:
will add them today :)